cmd/tracee-ebpf/main.go

```diff
@@ -209,6 +203,12 @@ func main() {
 				return fmt.Errorf("error creating Tracee: %v", err)
 			}
 
+			// environment capabilities
+			err = ensureCapabilities(OSInfo, &cfg, c.Bool(allowHighCapabilitiesFlag))
+			if err != nil {
+				return err
+			}
+
```

Comment:
Let's not change this in this documentation PR, but in a dedicated one

Reply:
Definitely. It was a letfover from my local branch!